docs/content/en/docs/getting-started/optional/cni.md

```diff
@@ -234,7 +212,7 @@ immediately install a CNI after uninstalling EKS Anywhere Cilium.
 {{% /alert %}}
 
 {{% alert title="Warning" color="warning" %}}
-Clusters created using the Full Lifecycle Controller prior to v0.15 that have removed the EKS Anywhere Cilium CNI must manually populate their `cluster.anywhere.eks.amazonaws.com` object with the following annotation to ensure EKS Anywhere does not attempt to re-install EKS Anywhere Cilium.
+Clusters created using the Full Lifecycle Controller prior to v0.15.0 that have removed the EKS Anywhere Cilium CNI must manually populate their `cluster.anywhere.eks.amazonaws.com` object with the following annotation to ensure EKS Anywhere does not attempt to re-install EKS Anywhere Cilium.
```

Comment:
I don't like using "Full Lifecycle Controller" term, that's really an implementation detail. I prefer, "Clusters created using Kubernetes API-compatible tooling such as kubectl, Terraform, or GitOps ..."